common/scripted_triggers/wc_leadership_triggers.txt

```diff
@@ -0,0 +1,107 @@
+# Created by: DukeJeans
+# Created on: 1/29/2019
+# Last Modified: 6/21/2019
+
+allows_archmagus_leader = {
+	is_magic_class_trigger = yes
+	NOT = { trait = archmagus_leader }
+}
+
+allows_offensive_caster_leader = {
+	is_magic_class_trigger = yes
+	NOT = { trait = offensive_caster_leader }
+}
+
+allows_defensive_caster_leader = {
+	is_magic_class_trigger = yes
+	NOT = { trait = defensive_caster_leader }
```

Comment:
So, you can get both defensive_caster_leader and offensive_caster_leader?
:thonk: